src/components/timeline.js

```diff
@@ -209,6 +215,11 @@ function Timeline({ durationInMilliseconds, parts, selectedPartIndex, onPartAdd,
   const renderSegment = (segment, index) => {
     const classes = classNames('Timeline-segment', { 'is-selected': index === selectedPartIndex });
     return (
+    // <div key={segment.key} className="Timeline-marker">
+    //   <div className="Timeline-markerTimecode">
+    //     abc
+    //   </div>
+    // </div>
```

Comment:
I guess this is not needed anymore :stuck_out_tongue_closed_eyes:

Reply:
Upf, indeed not! 😳